### DISCUSSION TITLE
pygalmesh rotation radius

### DISCUSSION THREAD

**Discussion by y-aoub** [nschloe/pygalmesh#146] [General]
Hi. How can I find the rotation radius variable in this code ? I have a polygon of a 2D bottle and I want to use rotation to make it 3D. Thank you.

```python
import pygalmesh

p = pygalmesh.Polygon2D([[0.5, -0.3], [1.5, -0.3], [1.0, 0.5]])
max_edge_size_at_feature_edges = 0.1
domain = pygalmesh.RingExtrude(p, max_edge_size_at_feature_edges)
mesh = pygalmesh.generate_mesh(
    domain,
    max_cell_circumradius=0.1,
    max_edge_size_at_feature_edges=max_edge_size_at_feature_edges,
    verbose=False,
)
```

**Comment by nschloe** [comment]
What do you mean by _rotation radius_?

**Reply by y-aoub** [reply]
I mean the distance between the rotation axis and the initial polygon.
I have this polygon of a bottle an I want to use rotation to make a 3D bottle. 
![bottle](https://user-images.githubusercontent.com/71386458/122640078-b6f8e400-d0fd-11eb-8980-4d0f0cad2eba.png)

**Reply by nschloe** [reply]
It always rotates around the z-axis, so you can control the radius by moving your 2D polygon around.

**Reply by y-aoub** [reply]
Ok, got it. Thanks.
